.github/workflows/promote_branch.yaml

```diff
@@ -3,14 +3,14 @@ name: Promote branch
 on:
   workflow_dispatch:
     inputs:
-      target-branch:
-        description: The name of the branch to be promoted
+      branches:
+        description: Which branch is being promoted where
         type: choice
         required: true
-        default: staging
+        default: development-to-staging
```

Comment:
development-to-staging isn't a branch. I don't think this makes things clearer. If you want to switch to options worded like this, I think the input name should change to something that makes sense

Reply:
My only goal is to make it clearer than it is today - the description says "The name of the branch to be promoted" - but that's kind of ambiguous: When you promote staging to production, are you promoting staging or are you promoting production?

My first idea was to just update the description, but what should it say? "The name of the branch to be promoted to"? That didn't sound great. So I thought spelling out what is promoted to where makes it crystal clear.

Do you have some other option in mind? Or what name instead of "branches" would be better if otherwise it looks good?

Reply:
Maybe `s/branches/promotion/`?

Reply:
I went with promotion-type, hope you like it, let me know. Renamed in a new commit.